skywalking-sniffer/skywalking-sdk-plugin/jedis-2.x-plugin/src/main/java/com/a/eye/skywalking/plugin/jedis/v2/define/JedisClusterInstrumentation.java

```diff
@@ -0,0 +1,97 @@
+package com.a.eye.skywalking.plugin.jedis.v2.define;
+
+import com.a.eye.skywalking.api.plugin.bytebuddy.AllObjectDefaultMethodsMatch;
+import com.a.eye.skywalking.api.plugin.interceptor.ConstructorInterceptPoint;
+import com.a.eye.skywalking.api.plugin.interceptor.InstanceMethodsInterceptPoint;
+import com.a.eye.skywalking.api.plugin.interceptor.enhance.ClassInstanceMethodsEnhancePluginDefine;
+import com.a.eye.skywalking.plugin.jedis.v2.JedisClusterConstructorWithListHostAndPortArgInterceptor;
+import com.a.eye.skywalking.plugin.jedis.v2.JedisMethodInterceptor;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+import java.util.Set;
+
+import static com.a.eye.skywalking.api.plugin.bytebuddy.ArgumentTypeNameMatch.takesArgumentWithType;
+import static net.bytebuddy.matcher.ElementMatchers.any;
+import static net.bytebuddy.matcher.ElementMatchers.not;
+import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
+
+/**
+ * {@link JedisClusterInstrumentation} presents that skywalking will intercept all constructors and methods of {@link redis.clients.jedis.JedisCluster}.
+ * There are two intercept classes to intercept the constructor of {@link redis.clients.jedis.JedisCluster}.
+ * {@link com.a.eye.skywalking.plugin.jedis.v2.JedisClusterConstructorWithHostAndPortArgInterceptor} intercepts all constructor with argument {@link redis.clients.jedis.HostAndPort}
+ * and the other constructor will intercept by class {@link JedisClusterConstructorWithListHostAndPortArgInterceptor}.
+ * {@link JedisMethodInterceptor} will intercept all methods of {@link redis.clients.jedis.JedisCluster}
+ *
+ * @author zhangxin
+ */
+public class JedisClusterInstrumentation extends ClassInstanceMethodsEnhancePluginDefine {
+
+    /**
+     * {@link redis.clients.jedis.HostAndPort} argument type name
+     */
+    private static final String ARGUMENT_TYPE_NAME = "redis.clients.jedis.HostAndPort";
+    /**
+     * Enhance class
+     */
+    private static final String ENHANCE_CLASS = "redis.clients.jedis.JedisCluster";
+    /**
+     * Class that intercept all constructors with arg.
```

Comment:
This comment make no sense. Which methods will be intercepted should be commented on the real class.